src/Stripe.net/Services/PaymentSourceService/PaymentSourceListOptions.cs

```diff
@@ -0,0 +1,11 @@
+namespace Stripe
+{
+    using Newtonsoft.Json;
+
+    public class PaymentSourceListOptions : ListOptions
+    {
+        /// <summary>Filter sources according to a particular object type.</summary>
+        [JsonProperty("object")]
+        public string Object { get; set; }
```

Comment:
We likely would have to show a list of enums since it's not documented anywhere right?

Reply:
We probably should, but the spec doesn't include a list of valid values (neither in the description string or by defining `object` as an enum).